recognition/advanced_tree_search.py

```diff
@@ -206,6 +190,8 @@ def __init__(
             self.config,
             self.post_config,
             self.lm_gc_job,
+            self.gc_job,
+            self.lm_image_jobs,
         ) = AdvancedTreeSearchJob.create_config(**kwargs)
```

Comment:
Is `.create_config` only called within this job? Otherwise we need to be careful with changing the returned variables. But I think you caught the cases here and otherwise the change is easy.

Reply:
the `AdvancedTreeSearchWithRescoringJob` below in this file inherits from `AdvancedTreeSearchJob` and uses `super().create_config`. that should be fixed.